release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -993,21 +1092,190 @@ If you are using the option `--keep-manifest-list=true`, the only valid value fo
+
+* Previously the Image Registry Operator did not use its entrypoint because an explicit command was provided. So a cluster-wide `trusted-ca` was not used by the Operator and the Operator could not connect to storage providers that do not work without custom `trusted-ca`. This fix removed the explicit command from the pod spec. Now the image entrypoint is used by the container that applies `trusted-ca`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1892799[*BZ#1892799*])
+
+* Previously the default log level for the pruner was `2`. So when an error happened, the pruner was dumping stack trace. This fix changed the default log level to `1`. Now only the error message is printed without stack traces. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1894677[*BZ#1894677*])
+
+* Previously the `configs.imageregistry.operator.openshift.io` status field did not update during the Operator sync, which meant the status field was not presenting the most up to date applied swift configuration. With this fix the sync process updates the `configs.imageregistry.operator.openshift.io` status to the spec values. The spec and status fields are in sync with the status field, presenting the applied configuration. link:https://bugzilla.redhat.com/show_bug.cgi?id=1907202[*BZ#1907202*])
+
+* Previously a lack of retries on a HTTP/2 protocol caused a related retryable error, which in turn caused mirroring to be cancelled with an error message. This fix added a retry when the error message corresponds to the HTTP/2 protocol related error. Now for these errors, the mirror operation is cancelled after attempting multiple times. link:https://bugzilla.redhat.com/show_bug.cgi?id=1907421[*BZ#1907421*])
+
+* Previously the absence of explicit user and group IDs on the `node-ca` daemon set confused the interpretation of what user and group were in use in the `node-ca` pods. This fix explicitly provides the `node-ca` daemon set with `runAsUser` and `runAsGroup` configuration. Now there is a clear definition of user and group when inspecting the `node-ca` DaemonSet YAML file. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1914407[*BZ#1914407*])
+
+*ImageStreams*
+
+* Previously, the image pruner did not account for images that were used by StatefulSets, Jobs, and Cronjob objects when it gathered lists of images that were in use. As a result, the wrong images could be pruned. The image pruner now accounts for images in use by these objects when it creates image lists. Images that are in use by these objects are no longer pruned. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1880068[*BZ#1880068*])
```

Comment:
s/StatefulSets, Jobs, and Cronjob objects/`StatefulSet`, `Job`, and `Cronjob` objects